pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -60,8 +60,29 @@ type ServiceBrokerSpec struct {
 	// CABundle is a PEM encoded CA bundle which will be used to validate a Broker's serving certificate.
 	// +optional
 	CABundle []byte `json:"caBundle,omitempty"`
+
+	// RelistBehavior specifies the type of relist behavior the catalog should exhibit
+	// when relisting ServiceClasses available from a broker
+	RelistBehavior ServiceBrokerRelistBehavior
```

Comment:
fields in versioned APIs need to have json tags associated with them.  For this field, it would be:

```go
RelistBehavior ServiceBrokerRelistBehavior `json:"relistBehavior"`
```

You'll need to add these to the fields in this file.